nextflow-genomics-cluster-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,334 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsNameForJumpBox": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Docker Virtual Machine (master node)."
+      }
+    },
+    "vmImageReference" :{
+      "type": "object",
+      "defaultValue": {
+        "publisher": "Canonical",
+        "offer": "UbuntuServer",
+        "sku": "16.04-LTS",
+        "version": "latest"
+      },
+      "metadata": {
+        "description": "The image to use for VMs created. This can be marketplace or custom image",
+        "link": "https://docs.microsoft.com/en-us/nodejs/api/azure-arm-compute/imagereference?view=azure-node-2.2.0"
+      }
+    },
+    "vmNodeSku": {
+      "type": "string",
+      "defaultValue": "Standard_F8s",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "vmMasterSku": {
+      "type": "string",
+      "defaultValue": "Standard_F16s",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "instanceCount": {
+      "type": "int",
+      "metadata": {
+        "description": "Number of cluster VM instances (100 or less)."
+      },
+      "defaultValue": 2,
+      "maxValue": 100
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username on all VMs."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password on all VMs."
+      }
+    },
+    "vnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the virtual network to deploy the scale set into."
+      },
+      "defaultValue": "nfvnet"
+    },
+    "subnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the subnet to deploy the scale set into."
+      },
+      "defaultValue": "nfsubnet"
+    },
+    "shareName": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure file share name."
+      },
+      "defaultValue": "sharedstorage"
+    },
+    "mountpointPath": {
+      "type": "string",
+      "defaultValue": "/datadisks/disk1",
+      "metadata": {
+        "description": "Path on VM to mount file shares. '/datadisks/disk1/' is a Premium Managed disk with high iops, this will suit most uses."
+      }
+    },
+    "nodeMaxCpus": {
+      "defaultValue": 2,
+      "type": "int",
+      "metadata": {
+        "description": "Sets the cluster.maxCpus setting on all cluster nodes"
+      }
+    }
+  },
+  "variables": {
+    "nextflowInitScript": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/nextflow-genomics-cluster-ubuntu/scripts/init.sh",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts

Reply:
Fixed these up.